torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -987,6 +987,22 @@ RegisterOperators reg(
                to_dispatch(self, device, scalarType, non_blocking, copy));
          },
          aliasAnalysisFromSchema()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA(
+             "aten::type.prim_dtype(Tensor(a) self, int? dtype=None, bool non_blocking=False, bool copy=False) -> Tensor(a|b)"),
```

Comment:
This is the exact same schema as `aten::to.prim_dtype(Tensor(a) self, int? dtype=None, bool non_blocking=False, bool copy=False`, and does the same thing. I think it would be simpler to add it by adding `(torch.type, "aten::to")` to the builtin operator registry here

Reply:
I agree. I tried looking earlier if I could just connect `type` -> `to` but couldn't find such an example.
Do you mean like this?
```
OperatorGenerator(torch.type, "aten::to"),
```